src/devices/Mhz19b/Mhz19b.cs

```diff
@@ -0,0 +1,218 @@
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Mhz19b"/> class.
+        /// </summary>
+        /// <param name="uartDevice">Path to the UART device / serial port, e.g. /dev/serial0</param>
+        public Mhz19b(string uartDevice)
+        {
+            if (uartDevice == null)
+            {
+                throw new ArgumentNullException(nameof(uartDevice));
+            }
+
+            // create serial port using the setting acc. to datasheet, pg. 7, sec. general settings
+            _serialPort = new SerialPort(uartDevice, 9600, Parity.None, 8, StopBits.One);
+            _serialPort.Encoding = Encoding.ASCII;
+            _serialPort.ReadTimeout = 1000;
+            _serialPort.WriteTimeout = 1000;
+        }
+
+        /// <summary>
+        /// Gets the current CO2 concentration from the sensor.
+        /// The validity is true if the current concentration was successfully read.
+        /// If the serial communication timed out or the checksum was invalid the validity is false.
+        /// If the validity is false the ratio is set to 0.
+        /// </summary>
+        /// <returns>CO2 concentration in ppm and validity</returns>
+        public (Ratio, bool) GetCo2Reading()
+        {
+            Ratio concentration = Ratio.FromPartsPerMillion(0);
+            bool validity = false;
+
+            try
+            {
+                var request = CreateRequest(Command.ReadCo2Concentration);
+                request[(int)MessageFormat.Checksum] = Checksum(request);
+
+                _serialPort.Open();
+                _serialPort.Write(request, 0, request.Length);
+
+                byte[] response = new byte[MessageSize];
+                if ((_serialPort.Read(response, 0, response.Length) == response.Length) && (response[(int)MessageFormat.Checksum] == Checksum(response)))
+                {
+                    concentration = Ratio.FromPartsPerMillion((int)response[(int)MessageFormat.DataHigh] * 256 + (int)response[(int)MessageFormat.DataLow]);
+                    validity = true;
+                }
+            }
+            finally
+            {
+                _serialPort.Close();
+            }
+
+            return (concentration, validity);
+        }
+
+        /// <summary>
+        /// Initiates a zero point calibration.
+        /// The sensor doesn't respond anything, so this is fire and forget.
+        /// </summary>
+        /// <returns>true, if the command could be send</returns>
+        public bool ZeroPointCalibration() => SendRequest(CreateRequest(Command.CalibrateZeroPoint));
+
+        /// <summary>
+        /// Initiate a span point calibration.
+        /// The sensor doesn't respond anything, so this is fire and forget.
+        /// </summary>
+        /// <param name="span">span value, e.g. 2000[ppm]</param>
+        /// <returns>true, if the command could be send</returns>///
+        public bool SpanPointCalibration(int span)
```

Comment:
Please use a verb for this action like InitiateSpanPointCalibration
Please also use the propre UnitsNet for the span. There is one for concentrations
Please add in the comments to minimum and maximum value and check it at the beginning of the function. From what I can see, it's from 0 to 65K

Reply:
I'll change this. A range check is reasonable as I don't expect that the sensor specification will change in later revision which could lead to inconsistency w/ the implementation.

Reply:
Done. Range check added. Exceed the range results in ArgumentException. Method renamed to PerformSpanPointCalibration. Parameter type changed to VolumnConcentration.